Build error: field 'kills' has incomplete type 'QMap<QString, int>'

Seems like you forgot `#include <QMap>` in world.h

```c++
make: *** [Makefile:1363: moc_world.o] Error 1
make: *** Waiting for unfinished jobs....
In file included from mainwindow.h:8,
                 from moc_mainwindow.cpp:10:
./world.h:109:25: error: field 'kills' has incomplete type 'QMap<QString, int>'
  109 |   QMap<QString, qint32> kills;
      |                         ^~~~~
In file included from /usr/include/qt/QtCore/qglobal.h:1296,
                 from /usr/include/qt/QtGui/qtguiglobal.h:43,
                 from /usr/include/qt/QtWidgets/qtwidgetsglobal.h:43,
                 from /usr/include/qt/QtWidgets/qmainwindow.h:43,
                 from /usr/include/qt/QtWidgets/QMainWindow:1,
                 from mainwindow.h:5,
                 from moc_mainwindow.cpp:10:
/usr/include/qt/QtCore/qtypeinfo.h:240:1: note: declaration of 'class QMap<QString, int>'
  240 | Q_DECLARE_MOVABLE_CONTAINER(QMap);
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~
make: *** [Makefile:1354: moc_mainwindow.o] Error 1
```

--- a/world.h
+++ b/world.h
@@ -4,6 +4,7 @@
 
 #include <QObject>
 #include <QRunnable>
+#include <QMap>
 #include "./worldinfo.h"
 #include "./worldheader.h"
 #include "./handle.h"

